module/Finna/src/Finna/View/Helper/Root/Markdown.php

```diff
@@ -39,41 +42,35 @@
 class Markdown extends \VuFind\View\Helper\Root\Markdown
 {
     /**
-     * Parsedown parser
-     *
-     * @var \Parsedown
+     * Constructor
      */
-    protected $parsedown = null;
-
-    /**
-     * Return HTML.
-     *
-     * @param string $markdown Markdown
-     *
-     * @return string
-     */
-    public function toHtml($markdown)
+    public function __construct()
```

Comment:
Tämän constructorin pitäisi ottaa samat parametrit kuin parentin. Sitten tuo converterin luonti pitäisi tehdä yliajetussa VuFind\Service\MarkdownFactory-luokassa. Eli contructorin voinee ottaa täältä kokonaan pois, kun on yliajettu `'League\CommonMark\MarkdownConverterInterface' => 'VuFind\Service\MarkdownFactory'`.

Reply:
Yliajoin view helperin factoryn, uskoisin että nää on nimenomaan muutoksia joita vain viewhelpperissä tarvitaan

Reply:
Jos ei haluta, että konfiguraatio vaikuttaa kuten VuFindin Markdown-factoryssa, niin sitten lienee ok. Onko näin?

Reply:
Joo, sille konfiguraatiolle ei mielestäni ole tarvetta